### PR TITLE
LAS-8: Property page

### PR DESCRIPTION
**Pull request by nickothan**

### CODE REVIEW

**Comment by revione** [comment]
All inside the project should have to be in english

**Comment by revione** [comment]
Follow this pattern
LAS-8: adding navigation

File: src/app/dashboard/property/navLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+import Link from "next/link";
+
+export function NavLinks() {
+  const pathname = usePathname();
+
+  return (
+    <nav className="flex gap-5">
```
Comment (revione) [inline src/app/dashboard/property/navLinks.tsx]: It should have to be an array

File: src/app/dashboard/property/page.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function Property() {
+  return <div>property</div>;
```
Comment (revione) [inline src/app/dashboard/property/page.tsx]: I dont see the image and the other information

File: src/app/dashboard/property/template.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
...fEodKvITJCvqLdIefUE6wpCsQPmKSqPGX3KiLr2XCdxIO65zmUc3RA1VZVO4C8LhcqEWXc+kNY2Myova3mqwXHVTFdOsdHlSzXwi1psbGqYpmnzVKK6mK6DxI71ElwkX7Lpg2lG/jm7BZ0XCmLhDtRO9VkLz+KlSbXVK24RG3KbQifcb5LoV1j/ABNZ4H/Eb8jzn0fmT6H91dNuVG5iowtdoR9lGPtYJPUIeFz/ADPKPN5Y6EmB9S0+i3rrgNBY3QGJ5wsb4et/4d73YplsNy0dOv6q1bdJJQ1Tsus2nGoov6dymGXKzzLpGZdJXjFWZmiZcIzK6zzLtMMu0jxlY5jQMrJhlVUFO7TVK7U5Yy8cxeMqIzXKopXKap11FxospWWAKjUpNdmRmgNrIoqJRyj8Q8OtxTqVKrgwEfMY8pgwWgxLui+LXt+GPe1jy9gcQ1+mNoORLdsoyX1/x1wo3NGA0uc04mjEWgnqAc18P4pZmm6CCD95rZgjcdiOTO4M6++KSrXpQXzugOcraWL6tsO186lerNB0SzioSV1MHqF5DGAhF3JcXgg4sR5b4Jh6k0BDwLoalcRdbDthcLGndBwrwQ0fk7UzTCspCsgLjFp1GfQNispCsllMaIaju2MtrIzXFK2mqPWSObuikcKqw4cV340bqsc481CUyTEcYrwXAuhzRG3PVUoKICjQlIu23HVGZV6hUIKmHFCmMlH4L43IH4lIXg5qhaUVhQcX8nJx+C9ZfJhl8TtCpaeqM45KbLxVou6d2U1Sujuqe3OicdopSm7otDDGrL6hX5OBTlG76rKUnFGtnHEM0jiysaVUjYMukw24jVZ6i4zqmqp0UGrdGiklZdC5B3VNxTwxbXDsb2Q7ctMSp0CnmlBycHsw6IzW6KK+8CWlSn8NrA0zJfq45REnZLv/AMObL4RptYQ4/wCoSXOnnmYC1IKnK7v5PkHp8fwfN+P/AOGNNtvNvjdXb1ADxnOWx7cl8/Phe5+JVpCi4vo4fiNbDi3H8o8pMyM8l+gqTjJzVTwtgFa8cAJJZJjMwHgSd4CtDqJpO9yGTpYNqtj5fZf4bXrx5mNZlPmcJ7EA5FZS8tTTe5jhDmktI6jIr7pwuofjNzOYdOesaTzXyn/ERoHEK0CMwfWNVpxZHKTTM+bFGEU0ZvAV3AuKQWjSZtRAhRhFKghpDqP/2Q=="
```
Comment (revione) [inline src/app/dashboard/property/template.tsx]: why is this commented?

File: src/app/dashboard/property/tenats/page.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export default function Tenats() {
+  return (
+    <div>
+      <h2>tenats</h2>
```
Comment (revione) [inline src/app/dashboard/property/tenats/page.tsx]: first letter uppercase!

**Review by revione — CHANGES_REQUESTED** [review]

File: src/app/dashboard/property/navLinks.tsx (hunk 1 above)
Comment (revione) [reply src/app/dashboard/property/navLinks.tsx]: ??

File: src/app/dashboard/property/page.tsx (hunk 2 above)
Comment (revione) [reply src/app/dashboard/property/page.tsx]: ??

File: src/app/dashboard/property/template.tsx (hunk 3 above)
Comment (revione) [reply src/app/dashboard/property/template.tsx]: ??

File: src/app/dashboard/property/tenats/page.tsx (hunk 4 above)
Comment (revione) [reply src/app/dashboard/property/tenats/page.tsx]: ??

**Review by revione — CHANGES_REQUESTED** [review]

File: src/app/dashboard/property/navLinks.tsx (hunk 1 above)
Comment (nickothan) [reply src/app/dashboard/property/navLinks.tsx]: ok

File: src/app/dashboard/property/template.tsx (hunk 3 above)
Comment (nickothan) [reply src/app/dashboard/property/template.tsx]: ok

File: src/app/dashboard/property/page.tsx (hunk 2 above)
Comment (revione) [reply src/app/dashboard/property/page.tsx]: do you have any question related to this?

File: src/app/dashboard/property/tenats/page.tsx (hunk 4 above)
Comment (revione) [reply src/app/dashboard/property/tenats/page.tsx]: do you have any question related to this?

**Pull request merged**